### ISSUE TITLE
Bug: "negative_prompts" not used when generating image

### ISSUE THREAD

**Issue by ejkkan** [Leonardo-Interactive/leonardo-ts-sdk#42]
When using `negative_prompt` with the API it is not being processed. 
You can test this either via the SDK or the API docs request.

It does make it very hard to use Leonardo as intended.

**Comment by ejkkan** [comment]
Anyone from the Leonardo team who could acknowledge this?

**Comment by amorphic** [comment]
The `negative_prompt` parameter is supported by the `generations` endpoint as per the [docs](https://docs.leonardo.ai/reference/creategeneration).

Can you please give more information as to how you've concluded that the parameter _"is not being processed"_?

**Comment by ejkkan** [comment]
Steps to reproduce: 
1. use "@leonardo-ai/sdk": "1.28.1", (newer version have bugs that breaks the sdk)
2. setup the client
3. use any config, including "negative_prompt"
4. watch the image details in Leonardo and you'll see that it is missing a negative prompt

The sdk seems to expect a "negativePrompt" variables, but the API does not accept that.

**Comment by alexadrake** [comment]
Hey @ejkkan can you provide a reproduceable code example (how you setup the SDK and make the request)?

**Comment by ejkkan** [comment]
@anuraagnalluri 
I've created this example for most of the issues I've reported. [See here](https://github.com/ejkkan/leonardo-ai-template/tree/master) 
So please config it to make the setup relevant for each issue.
Notice that I'm using v1.28 specifically because of functionality being broken [as mentioned in](https://github.com/Leonardo-Interactive/leonardo-ts-sdk/issues/39)

**Comment by speakeasybot** [comment]
Hi Erik, this should be addressed in SDK release `1.44.1` https://github.com/Leonardo-Interactive/leonardo-ts-sdk/releases/tag/v1.44.1 ? That should also address the regression linked

**Comment by ejkkan** [comment]
Still broken in `1.44.1`.
If you update the SDK in the example repo I provided you should see the faulty behavior.

**Comment by alexadrake** [comment]
Hi @ejkkan , I used the script you provided  and did a fresh install of `1.44.1` and it worked as expected. One possible explanation is that the dependencies may not actually be updated, in which case deleting `node_modules` and doing a fresh install may resolve the issue.

Note that I had to fix the script to call `GenerateImages` instead of the inline script and I had to change the request fields to `camelCase`.
